src/SimplePolyModel.cc

```diff
@@ -78,7 +78,7 @@ SimplePolyModel::SimplePolyModel(const CcdImageList &L,
 const Mapping* SimplePolyModel::getMapping(const CcdImage &C) const
 {
   mapType::const_iterator i = _myMap.find(&C);
-  if  (i==_myMap.end()) throw LSST_EXCEPT(pex::exceptions::InvalidParameterError,"SimplePolyModel::GetMapping, never heard of CcdImage "+C.Name());
+  if  (i==_myMap.end()) throw LSST_EXCEPT(pex::exceptions::InvalidParameterError,"SimplePolyModel::GetMapping, never heard of CcdImage "+C.getName());
```

Comment:
Technically only guaranteed to compare equal in C++14 (const vs non-const iterator). But we ar nitpicking here.